htdocs/projet/element.php

```diff
@@ -1288,7 +1288,7 @@
 				print "</td>\n";
 
 				// Ref
-				print '<td class="left nowraponall tdoverflowmax250">';
+				print '<td class="left nowraponall">';
```

Comment:
We need the tdoverflowmax250 for the case of object that do have a download arrwo and output a very long ref including a text (like loan , tasks). Don't know the liste exact so i suggest to remove it for all excet if 
$element instanceof Task is true or $element instanceof Loan is true

Reply:
I have put a table inside the td to prevent adding another td because the table head would not be right.